cmd/errtrace/main.go

```diff
@@ -281,30 +281,40 @@ func goListFiles(patterns []string) (files []string, err error) {
 	}
 
 	if err := cmd.Start(); err != nil {
-		return nil, fmt.Errorf("start: %w", err)
+		return nil, fmt.Errorf("start go list: %w", err)
```

Comment:
FYI these error messages don't need to say "go list"; the caller already does that.

Reply:
Fixed.